content/2.application.md

```diff
@@ -55,3 +55,12 @@ To simplify the development of a custom Payments CorDapp in the absence of the G
 
 ### PSP payments gateway
 The PSP payments gateway is an endpoint of a proprietary off-chain payments solution on the PSP side.
+
+### Supplementary parts of the Payments SDK
+
+1. [ISO20022 XML](https://github.com/gevamu/corda-payments-sdk/tree/master/iso20022)
+2. Payments App Sample:
+    - [Payments App Backend - Gevamu](https://github.com/gevamu/corda-payments-sdk/tree/master/payments-app-sample/payments-app-backend)
+    - [Payments App Frontend - Client App](https://github.com/gevamu/corda-payments-sdk/tree/master/payments-app-sample/payments-app-frontend)
+3. [Payments-Contracts](https://github.com/gevamu/corda-payments-sdk/tree/master/payments-contracts)
+4. [Payments-Workflows](https://github.com/gevamu/corda-payments-sdk/tree/master/payments-workflows)
```

Comment:
These are 2 parts of SDK. I would group these points and place it on the first place.

Reply:
Fixed